main.py

```diff
@@ -20,9 +17,9 @@
 
 # Note: Replace with your onboarded log types
 onboarded_log_types = [
-    PantherLogType.AWS_CloudTrail,
-    PantherLogType.AWS_GuardDuty,
-    PantherLogType.Okta_SystemLog,
+    LogType.AWS_CloudTrail,
```

Comment:
Shouldn't these be updated to the new log format?

Reply:
The new log stuff hasn't been published yet because the versioning stuff is broken. So this is just upgrading to 0.1.1a35. The log stuff will probably be in a36